packages/core/strapi/src/cli/index.ts

```diff
@@ -0,0 +1,74 @@
+import { Command } from 'commander';
+
+import { commands as strapiCommands } from './commands';
+
+import { createLogger } from './utils/logger';
+import { loadTsConfig } from './utils/tsconfig';
+import { CLIContext } from './types';
+
+const createCLI = async (argv: string[], command = new Command()) => {
+  try {
+    // NOTE: this is a hack to allow loading dts commands without make dts a dependency of strapi and thus avoiding circular dependencies
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const dtsCommands = require(require.resolve('@strapi/data-transfer')).commands;
```

Comment:
Do we still need this since data-transfer now depends on `core/core` & not `core/strapi`? (or is it?)

Reply:
Yes we still need to load the dts commands in the cli. but now data-transfer and strapi/strapi are not in a circle